model_security_adjust_sst/security/hr_security.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="group_hr_operator" model="res.groups">
+        <field name="name">Operator</field>
+        <field name="category_id" ref="base.module_category_human_resources"/>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+    </record>
+
+    <record id="hr.group_hr_user" model="res.groups">
+        <field name="implied_ids"
+               eval="[(3, ref('base.group_user')), (4, ref('group_hr_operator'))]"/>
```

Comment:
Doesn't this work?
```.xml
<field name="implied_ids" eval="[(6, ref('group_hr_operator'))]"/>
```

Reply:
Yes, it is right. We could have this for groups